locale/Makefile.am

```diff
@@ -20,55 +20,107 @@
 nobase_localedata_DATA = \
```

Comment:
I believe that CLDR has more than these? I'd say put all emojis files, even if we don't have a corresponding NVDA file.